spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/AutoConfigurationSorterTests.java

```diff
@@ -17,11 +17,7 @@
 package org.springframework.boot.autoconfigure;
 
 import java.io.IOException;
-import java.util.Arrays;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Properties;
-import java.util.Set;
+import java.util.*;
```

Comment:
We don't use wildcard imports. Please revert this change

Reply:
oh,sorry!It is my carelessness.I will correct it immediately.